### ISSUE TITLE
Device freezes after the double pause feedback

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#221]
```
What steps will reproduce the problem?

1. Open the global context menu.
2. pause feedback .
3 .Lock the device.
4. Unlock it.
5. Open the global context menu.
6. pause feedback  again.

What is the expected output? What do you see instead?

TalkBack freezing device. I totally lose control over the device.

What version of the product are you using? On what operating system?

TalkBack 3.3.1, Android 4.2.2, Galaxy Nexus.

```

Original issue reported on code.google.com by `electrog...@gmail.com` on 12 Apr 2013 at 11:08

**Comment by GoogleCodeExporter** [comment]
Original comment by `caseybur...@google.com` on 14 Apr 2013 at 11:59
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.3.2 Beta 1
```

Original comment by `caseybur...@google.com` on 20 Apr 2013 at 6:58
- Changed state: **Fixed**
